### PR TITLE
[Conversion gRPC-REST] Statistic endpoint

### PR DESCRIPTION
**Pull request by pauloabreu**
## Resume
This PR creates a new REST endpoint called Statistic.

This endpoint this data above of a org:
```
active_flows 
active_classifiers
active_contacts
```

### CODE REVIEW

File: weni/internal/statistic/tests.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+class TembaRequestMixin(ABC):
+    def reverse(self, viewname, kwargs=None, query_params=None):
+        url = reverse(viewname, kwargs=kwargs)
+
+        if query_params:
+            return "%s?%s" % (url, urlencode(query_params))
+        else:
+            return url
```
Comment (elitonzky) [inline weni/internal/statistic/tests.py:21]: There would be no need for "else" in this case, you could write it as follows
 
  if query_params:
      return "%s?%s" % (url, urlencode(query_params))

  return url

File: weni/internal/statistic/views.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+from weni.internal.views import InternalGenericViewSet
+
+
+class StatisticEndpoint(RetrieveModelMixin, InternalGenericViewSet):
+    lookup_field = "uuid"
+
+    def retrieve(self, request, *args, **kwargs):
+        params = self.request.query_params
+
```
Comment (elitonzky) [inline weni/internal/statistic/views.py:16]: Remove empty line

File: weni/internal/urls.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+
+internal_urlpatterns = []
+internal_urlpatterns += orgs_urls
+internal_urlpatterns += flows_urls
+internal_urlpatterns += users_urls
+internal_urlpatterns += tickets_urls
+internal_urlpatterns += statistics_urls
+
+urlpatterns = [path("internals/", include(internal_urlpatterns))]
```
Comment (elitonzky) [inline weni/internal/urls.py:27]: missing line break at end of code

**Review by elitonzky — COMMENTED** [review]
small corrections

File: weni/internal/urls.py (hunk 3 above)
Comment (pauloabreu) [reply weni/internal/urls.py:27]: Solved!

**Review by BarbosaJackson — APPROVED** [review]

**Pull request merged**